breadcord/helpers.py

```diff
@@ -143,10 +143,11 @@ def decorator(func: ItemCallbackType[V, Button[V]]) -> ItemCallbackType[V, Butto
         if not inspect.iscoroutinefunction(func):
             raise TypeError('button function must be a coroutine function')
 
+        max_id_length = 100
         # noinspection PyUnresolvedReferences
         custom_id = f'{func.__module__}:{func.__qualname__}'.removeprefix('breadcord.')
-        if len(custom_id) > 100:
-            raise RuntimeError(f'decorated function path exceeds 100 characters: {custom_id}')
+        if len(custom_id) > max_id_length:
+            raise RuntimeError(f'decorated function path exceeds {max_id_length} characters: {custom_id}')
```

Comment:
Nitpick - constants should be UPPER_SNAKE_CASE

```suggestion
        max_id_length = 100
        # noinspection PyUnresolvedReferences
        custom_id = f'{func.__module__}:{func.__qualname__}'.removeprefix('breadcord.')
        if len(custom_id) > max_id_length:
            raise RuntimeError(f'decorated function path exceeds {max_id_length} characters: {custom_id}')
```

Reply:
Not inside a function. PyCharm complains.